spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -265,6 +265,7 @@
             complete_doc_auth_steps_before_document_capture_step
 
             expect(page).to have_current_path(idv_document_capture_url)
+            expect(max_capture_attempts_before_native_camera).to eq('65535')
```

Comment:
Where did this number come from - I feel like for an "arbitrarily high number" it's usually some set of nines like "9999" or something.

Reply:
@charleyf 65535 is the unsigned maximum of 16bit integer.

Reply:
Unfortunately I did not find a constant like` FixNum.MAX` in `Ruby`

Reply:
> In original excel you can have maximum 65535 rows of data.

TIL!!!

Reply:
I saw `Float::INFINITY` being used [elsewhere in the code base](https://github.com/18F/identity-idp/blob/main/app/components/manageable_authenticator_component.html.erb#L44).  It's not an int/integer, but it could be worth a try?

We could also define this arbitrarily high number somewhere, if there's a desire to avoid magic numbers.

Reply:
@eileen-nava, that value need to be a number since it will be used by javascript. Changed to some other build in constants now.